### PR TITLE
exact steps to spin up a test cluster for dev

### PR DESCRIPTION
**Pull request by uwedeportivo**
for devs new to k8s or unfamiliar with our setup

### CODE REVIEW

File: handbook/engineering/deployments.md [hunk 1]
```diff
@@ -88,6 +88,62 @@ The other clusters are deployed and rolled back in the same way as sourcegraph.c
...
 	```
+ 
+ ## How to start a test cluster in our "Sourcegraph Auxiliary' project on GCP
+ 
+- Go to [Sourcegraph Auxiliary](https://console.cloud.google.com/kubernetes/list?project=sourcegraph-server)
+- Click create a cluster.
+- Give it a name (a good convention is to prefix with your username).
+- Keep the defaults zonal and us-central1.
+- Set the default pool to 3 nodes.
+- Change machine type to n1-standard-8.
```
Comment (mrnugget) [inline handbook/engineering/deployments.md]: Recommendation in [deploy-sourcegraph here]](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/k8s.md) is n1-standard-16 — does `n1-standard-8` also work?

File: handbook/engineering/deployments.md [hunk 2]
```diff
@@ -88,6 +88,62 @@ The other clusters are deployed and rolled back in the same way as sourcegraph.c
...
+- Click "Create Cluster".
+- When cluster is ready, click connect and copy/paste the command and execute it (it looks something 
+  like `gcloud container clusters get-credentials ....`). Now kubectl is acting on this cluster.
+- Give yourself admin superpowers by executing: 
+
+```shell script
```
Comment (mrnugget) [inline handbook/engineering/deployments.md]:
```suggestion
```shell
```

File: handbook/engineering/deployments.md [hunk 3]
```diff
@@ -88,6 +88,62 @@ The other clusters are deployed and rolled back in the same way as sourcegraph.c
...
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user $(gcloud config get-value account)`
```
Comment (mrnugget) [inline handbook/engineering/deployments.md]:
```suggestion
kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user $(gcloud config get-value account)
```

File: handbook/engineering/deployments.md [hunk 4]
```diff
@@ -88,6 +88,62 @@ The other clusters are deployed and rolled back in the same way as sourcegraph.c
...
+```
+
+- cd to your clone of [deploy-sourcegraph](https://github.com/sourcegraph/deploy-sourcegraph) and follow the remaining
+steps of the [installation](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/install.md).
+
+```shell script
+./kubectl-apply-all.sh
+```
+
+> Recommendation: [k9s](https://github.com/derailed/k9s) is a nice command-line GUI tool for common kubectl operations.
```
Comment (mrnugget) [inline handbook/engineering/deployments.md]: Thank you!

File: handbook/engineering/deployments.md [hunk 5]
```diff
@@ -88,6 +88,62 @@ The other clusters are deployed and rolled back in the same way as sourcegraph.c
...
+> It shows the state of your cluster and offers keyboard short-cuts for all the common kubectl commands.
+
+- Once all the pods are running you can port-forward the frontend (or any other services you are interested in)
+
+```shell script
+kubectl port-forward svc/sourcegraph-frontend 3080:30080 
+```  
+
+Please delete your test cluster when you are done testing by going
```
Comment (mrnugget) [inline handbook/engineering/deployments.md]:
```suggestion
Please delete your test cluster when you are done testing by going to
```

File: handbook/engineering/deployments.md (hunk 1 above)
Comment (uwedeportivo) [reply handbook/engineering/deployments.md]: changed to -16

depending on what you're testing -8 works but it's simpler to give clear instructions

**Pull request merged**